Installed .jar is missing some classes

Steps.

1. I've checked out this repo.
2. I've done a ./gradlew install
3. I can see the installed dependency:
`.m2/repository/com/authenteq/java-crypto-conditions/0.1/java-crypto-conditions-0.1.jar`

Shouldn't this be called `java-bigchaindb-driver`? I saw [this](https://github.com/authenteq/java-bigchaindb-driver/blob/1855b68ca1c232a73ca841473f13d656876af1d3/settings.gradle#L1).

I import the dependency in a maven project:

```xml
        <dependency>
            <groupId>com.authenteq</groupId>
            <artifactId>java-crypto-conditions</artifactId>
            <version>0.1</version>
        </dependency>
```

I'm not able to create a transaction, I get the following error:
```java
Exception in thread "main" java.lang.NoClassDefFoundError: org/interledger/cryptoconditions/types/Ed25519Sha256Condition
	at com.authenteq.BigchaindbTransaction.buildTransactionJson(BigchaindbTransaction.java:83)
	at com.authenteq.BigchaindbTransaction.<init>(BigchaindbTransaction.java:59)
	at com.rezzonade.persistence.bigchaindb.Example.main(Example.java:52)
Caused by: java.lang.ClassNotFoundException: org.interledger.cryptoconditions.types.Ed25519Sha256Condition
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 3 more
```

I can solve this by manually installing `java-crypto-conditions-2.0.0-SNAPSHOT.jar` by using:

```sh
mvn install:install-file -Dfile=yourClonedRepo/libs/java-crypto-conditions-2.0.0-SNAPSHOT.jar -DgroupId=org.interledger -DartifactId=java-crypto-conditions -Dversion=2.0.0-SNAPSHOT -Dpackaging=jar
```

Am I missing something or are these actual bugs?

--- a/settings.gradle
+++ b/settings.gradle
@@ -1,1 +1,1 @@
-rootProject.name = 'java-crypto-conditions'
+rootProject.name = 'java-bigchaindb-driver'

